test/k8s/assertion_helpers.go

```diff
@@ -229,13 +228,6 @@ func DeployCiliumOptionsAndDNS(vm *helpers.Kubectl, ciliumFilename string, optio
 	}
 }
 
-// SkipIfBenchmark will skip the test if benchmark is not specified
-func SkipIfBenchmark() {
-	if !config.CiliumTestConfig.Benchmarks {
```

Comment:
Looks like this is the only use of the `CiliumTestConfigType.Benchmarks` field. I think we could drop the following as well:

https://github.com/cilium/cilium/blob/72374ab9c5f1bcc22a87babed5a76de9c015d1e7/test/config/config.go#L43

and

https://github.com/cilium/cilium/blob/72374ab9c5f1bcc22a87babed5a76de9c015d1e7/test/config/config.go#L96-L97

Reply:
Good catch! Hopefully I haven't opened Pandora's box :sweat_smile: